docs/DeveloperGuide.md

```diff
@@ -388,6 +938,7 @@ Use case ends.
 2. Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands)
    should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. Should be 
 
 {More to be added}
 
```

Comment:
Glossary can be updated. e.g. "Private Contact Detail" is no longer relevant to the task manager. Think about what are the terms used by the task manager that may not be obvious to developers, e.g. "task" of different kind etc.